bluemix/terminal/table_test.go

```diff
@@ -0,0 +1,29 @@
+package terminal_test
+
+import (
+	"bytes"
+	"github.com/stretchr/testify/assert"
+	"testing"
+
+	. "github.com/IBM-Cloud/ibm-cloud-cli-sdk/bluemix/terminal"
+)
+
+func TestPrintTableSimple(t *testing.T) {
```

Comment:
Thanks for adding these tests. Though I want to ask if you can maybe add some negative tests? What happens if the headers are empty strings ? Just want to make sure this new functionality is robust.